recipes/stdio-mgr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "stdio-mgr" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bskinn/stdio-mgr/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b70044b1b31b9be3935d0b1e3ada3d713f7274d2124ffff42195e1fdac0a4029
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - attrs >=17.1
```

Comment:
This is probably not needed at build time, unless the package imports itself. Can you try to remove it and see how it goes?

```suggestion
```

Reply:
Agreed, attrs should not be needed at build.

Reply:
Ah, nope, was thinking this was a different package.

v1.0.1 here *does* import the package at build, to retrieve the version number. One of many overdue points of needed maintenance.

Reply:
Sure - I overlooked this.  Let me push a fix.